osu.Server.Queues.ScorePump/Queue/ImportHighScores.cs

```diff
@@ -309,7 +309,7 @@ public async Task Run(HighScore[] scores)
                             RulesetID = ruleset.RulesetInfo.OnlineID,
                             Passed = true,
                             TotalScore = (int)referenceScore.TotalScore,
-                            Accuracy = referenceScore.Accuracy,
+                            Accuracy = Math.Round(referenceScore.Accuracy, 4, MidpointRounding.ToEven),
```

Comment:
Same as https://github.com/ppy/osu/pull/20847#discussion_r1002085986 to some extent.